tests/callbacks/test_tqdm_progress_bar.py

```diff
@@ -81,9 +81,6 @@ def test_tqdm_progress_bar_totals(tmpdir):
 
     trainer = Trainer(default_root_dir=tmpdir, max_epochs=1)
     bar = trainer.progress_bar_callback
-    assert float("inf") == bar.total_train_batches
-    assert 0 == bar.total_val_batches
-    assert 0 == bar.total_test_batches
```

Comment:
Unfortunately we lose this functionality since these props can only be accessed once the `setup` hook has been called. Is this ok?